### PR TITLE
0.34 minor fixes

### PR DESCRIPTION
**Pull request by albi3ro**
Just a couple minor documentation fixes.

### CODE REVIEW

File: pennylane/ops/op_math/controlled.py [hunk 1]
```diff
@@ -109,7 +111,7 @@ def workflow(theta, w, cw):
             def func(arg):
               qml.RX(theta, wires=arg)
 
-            @qml.cond(theta > 0.0)
+            @catalyst.cond(theta > 0.0)
```
Comment (timmysilv) [inline pennylane/ops/op_math/controlled.py:114]: has the catalyst team confirmed that we're ok with advertising it this way? I know you spotted an issue using `qml.cond`, so I just wanna make sure

**Review by timmysilv — COMMENTED** [review]
PR 5000!

File: pennylane/ops/op_math/controlled.py (hunk 1 above)
Comment (albi3ro) [reply pennylane/ops/op_math/controlled.py:114]: Yes. This is the one way to get this code to work, as `catalyst.cond`, `qml.cond` + qjit, and `qml.cond` all have different signatures and behaviours. 

We could probably do a bigger rewrite to this example instead, but this more or less works.

Comment (albi3ro) [reply pennylane/ops/op_math/controlled.py:114]: @maliasadi

Comment (albi3ro) [reply pennylane/ops/op_math/controlled.py]: change reverted, as is fixed in PR #5006

**Review by timmysilv — APPROVED** [review]
🎉

**Pull request merged**